src/cysignals/cysignals_config.h.in

```diff
@@ -27,17 +27,24 @@
 
 
 /* An implementation of atomic variables might not be allowed with OpenMP */
-#if _OPENMP
-#define CYSIGNALS_C_ATOMIC CYSIGNALS_C_ATOMIC_WITH_OPENMP
-#define CYSIGNALS_CXX_ATOMIC CYSIGNALS_CXX_ATOMIC_WITH_OPENMP
-#define CYSIGNALS_STD_ATOMIC CYSIGNALS_STD_ATOMIC_WITH_OPENMP
-#endif
-
+#undef CYSIGNALS_C_ATOMIC_WITH_OPENMP
+#undef CYSIGNALS_CXX_ATOMIC_WITH_OPENMP
+#undef CYSIGNALS_STD_ATOMIC_WITH_OPENMP
 
 /* Which implementation of atomic variables (if any) to use? */
 #ifndef __cplusplus
+
+#if defined(CYSIGNALS_C_ATOMIC_WITH_OPENMP) || !_OPENMP
```

Comment:
What happens with `!_OPENMP` if `_OPENMP` is not defined?  I'm honestly not sure how that's supposed to work with the C preprocesser.

Reply:
`!_OPENMP` works as expected. I checked. However, I agree that it is not clean. And maybe some compilers are more strict about this.

Reply:
And there are other places in cysignals to fix this. I'll fix it and open a pull request.